http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcClientConfiguration.java

```diff
@@ -651,4 +722,99 @@ public String getTokenSignatureAlgorithm() {
         return tokenSignatureAlgorithm;
     }
 
+    public String getAuthenticationRequestFormat() {
+        return authenticationRequestFormat;
+    }
+
+    public void setAuthenticationRequestFormat(String requestObjectType ) {
+        this.authenticationRequestFormat = requestObjectType;
+    }
+
+    public String getRequestSignatureAlgorithm() {
+        return requestSignatureAlgorithm;
+    }
+
+    public void setRequestSignatureAlgorithm(String algorithm) {
+        this.requestSignatureAlgorithm = algorithm;
+    }
+
+    public String getRequestEncryptAlgorithm() {
+        return requestEncryptAlgorithm;
+    }
+
+    public void setRequestEncryptAlgorithm(String algorithm) {
+        this.requestEncryptAlgorithm = algorithm;
+    }
+
+    public String getRequestEncryptEncValue() {
+        return requestEncryptEncValue;
+    }
+
+    public void setRequestEncryptEncValue (String enc) {
+        this.requestEncryptEncValue = enc;
+    }
+
+    public String getRealmKey () {
+        return realmKey;
+    }
+
+    public void setRealmKey(String key) {
+        this.realmKey = key;
+    }
+
+    public String getClientKeyStoreFile () {
+        return clientKeyStoreFile;
+    }
+
+    public void setClientKeyStoreFile(String keyStoreFile) {
+        this.clientKeyStoreFile = keyStoreFile;
+    }
+
+    public String getClientKeyStorePassword () {
+        return clientKeyStorePass;
+    }
+
+    public void setClientKeyStorePassword(String pass) {
+        this.clientKeyStorePass = pass;
+    }
+
+    public String getClientKeyPassword () {
+        return clientKeyPass;
+    }
```

Comment:
These client keystore methods/variables will need to be renamed based on the discussion in the analysis doc.

Reply:
updated all names related to keystores.